sdk/tables/Azure.Data.Tables/tests/perf/Infrastructure/ComplexPerfEntityGenerator.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System;
```

Comment:
nit:  You may want to consider adding a blank line for consistency with other files.

Reply:
After looking the PR over, I'm not sure this is used since the base class has a generate method....